packages/smooth_app/lib/helpers/analytics_helper.dart

```diff
@@ -407,6 +421,19 @@ class AnalyticsHelper {
         productType: product.productType ?? ProductType.food,
       );
 
+  static void trackChangedPreferences(
+    AnalyticsChangedPreferences event, {
+    final String? attributeId,
+    final String? importanceId,
+  }) =>
+      trackCustomEvent(
+        event.name,
+        AnalyticsCategory.changedPreferences.tag,
+        action: attributeId == null || importanceId == null
+            ? null
+            : '$attributeId=$importanceId',
```

Comment:
we don't want to be in the business of logging individual preferences.

Reply:
From what I understand, we need an event when we exit the food preferences and there are some changes.
In the event, we only list prefs that are different from the default state.

Reply:
@g123k Then I guess we'll be better off with a new PR from scratch.